spring-cloud-starter-app-broker-logging/build.gradle

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2002-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+description = "Spring Cloud App Broker Starter for Logging"
+
+dependencyManagement {
+	imports {
+		mavenBom "org.springframework.boot:spring-boot-dependencies:${springBootVersion}"
+	}
+}
+
+dependencies {
+	compile project(":spring-cloud-starter-app-broker")
+	compile project(":spring-cloud-app-broker-logging")
```

Comment:
Could we use `api` instead of `compile`?

Reply:
I'm going to refactor the entire gradle build to use the new maven publishing plugin soon and I'll change everything to api/integration at that point.